easybuild/framework/easyblock.py

```diff
@@ -1636,6 +1636,28 @@ def guess_start_dir(self):
         change_dir(self.start_dir)
         self.log.debug("Changed to real build directory %s (start_dir)", self.start_dir)
 
+    def check_accepted_eula(self, more_info=None):
+        """Check whether EULA for this software is accepted in current EasyBuild configuration."""
+
+        accepted_eulas = build_option('accept_eula') or []
+        if self.cfg['accept_eula'] or self.name in accepted_eulas:
```

Comment:
For ease of use, shouldn't the self.name in accepted_eulas be a case insensitive match?

Reply:
I'm in favor of keeping it strict actually.

You don't want to accidentally accept a EULA either, imagine the consequences... ;)

Reply:
ok.